src/medCore/data/medAbstractDataWriter.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MEDABSTRACTDATAWRITER
+#define MEDABSTRACTDATAWRITER
+
+#include <dtkCore/dtkAbstractDataWriter.h>
+#include <medCoreExport.h>
+
+
+class medAbstractData;
+/**
+ * Extending dtkAbstractDataWriter class to allow the export of a list of data
+ */
+class MEDCORE_EXPORT medAbstractDataWriter : public dtkAbstractDataWriter
+{
+    Q_OBJECT
+
+public:
+    using dtkAbstractDataWriter::setData;
+    virtual void setData(QList<medAbstractData*> data) {}
```

Comment:
I know this has been merged but I just realised this while reviewing the NavX exporter PR. Why didn't you store the dataList directly in ```medAbstractDataWriter``` and give access to it with a method called ```dataList``` (or similar). This is what ```dtkAbstractDataWriter``` does for single data. Without this, every exporter that implements this will have to stow it itself which will just mean unnecessary code duplication.